source/includes/_neighborly_location_review_details.md

```diff
@@ -77,4 +78,30 @@ curl -L -X GET 'https://api.gcion.com/apgb2b-reporting/client_reports/neighborly
 |total_pages|Integer|How many pages are there left. |
 |details|Array|Review details for specified locations. [Details Object](https://api2-test-unifiedsyncplatform.dwyergroup.com/swagger/index.html)|
 
+### POST
 
+Use POST with a JSON payload to create/update Neighborly location for a given advertiser.
+
+#### Examples
+
+```
+curl --location --request POST 'http://localhost:3001/client_reports/neighborly/location' \
```

Comment:
The documentation examples should have the most likely URL that will be used by the caller.  So for our public APIs you will see https://api.localiqservices.com as the endpoint.  But for internal, we want to gently guide callers to our internal URLs like https://data-connect-prod.gannettdigital.com/client_reports/neighborly/location and not the https://api.gcion.com/apgb2b-reporting/client_reports/neighborly/location URL used earlier in this file and never http://localhost:3001/client_reports/neighborly/location (that's just you and I).

Please update ALL the examples in this file (even the gcion ones) to point at https://data-connect-prod.gannettdigital.com/client_reports/neighborly/location and make no mention of an x-api-key (which is no longer needed when using internal URLS).